ChessStatePattern/src/states/State.java

```diff
@@ -1,16 +1,36 @@
 package states;
 
 import main.*;
+import states.commons.Move;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public abstract class State {
 	
 	ChessGame game;
-	
+	String activePlayer;
```

Comment:
Seems like you added `activePlayer` there and plan to track each player's actions,
but we need to ensure that each time switch the player action, this `activePlayer` needs to be updated. Will it be implemented in `setActivePlayer `?